src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes string with styles
- * and returns object where CSS properties are keys
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
+ * and returns an object where CSS properties are keys
  * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const rows = sourceString.split(/\r?\n/);
+
+  rows.forEach((row) => {
+    if (row === '' || !row.includes(':')) {
+      return;
+    }
+
+    const property = row.split(/:|;/)
+      .map(element => element.trim());
+
+    result[property[0]] = property[1];
```

Comment:
This could potentially assign an empty value to the result object if there are multiple ':' or ';' in the row. It would be better to ensure the property array has at least two elements before assigning.